### ISSUE TITLE
Update License copyright year to 2018

### ISSUE THREAD

**Issue by SD10** [Moya/Moya#1522]
We need to update the copyright year in [LICENSE.md](https://github.com/Moya/Moya/blob/aa33a0466d7333516691b3e5212c5acb41990928/License.md) to 2018. This is a great first issue!

```diff
-- Copyright (c) 2017 Artsy, Ash Furrow
++ Copyright (c) 2018 Artsy, Ash Furrow
```

**Comment by SD10** [comment]
@richgabrielli if you want more forking & PR practice this is another great issue 🤣 🤣 No pressure!

**Comment by AndrewSB** [comment]
Another good pattern I've seen is along the lines of 

```
Copyright 2004-present Artsy, Ash Furrow
```

where 2004 is the year when it started. That way we don't have to keep it up to date

**Comment by SD10** [comment]
@AndrewSB That’s also an option. 2014 you mean!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by richgabrielli** [comment]
My $0.02: "...2014-present.." is a great fixit and forget it solution.

**Comment by SD10** [comment]
I’m down with that!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by SD10** [comment]
Closed by #1523
